src/scenes/Login/index.tsx

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
-import { sleep } from '../../util';
+import { LoginMutationResult, useLoginMutation } from '../../api';
 import { LoginForm } from './LoginForm';
 
 export const Login = () => {
+  const [login] = useLoginMutation();
+
   return (
     <LoginForm
       onSubmit={async (data) => {
-        await sleep(1000);
-        alert(JSON.stringify(data, undefined, 2));
+        try {
+          const res: LoginMutationResult['data'] = await login({
```

Comment:
you shouldn't need to explicitly define the return type...
```suggestion
          const res = await login({
```

Reply:
Whoops! I was debugging something and this shouldn't have made it in.